package.json

```diff
@@ -164,6 +164,7 @@
     "onCommand:extension.NAMESPACE(chrome-debug).addCustomBreakpoints",
     "onCommand:extension.NAMESPACE(chrome-debug).removeAllCustomBreakpoints",
     "onCommand:extension.NAMESPACE(chrome-debug).removeCustomBreakpoint",
+    "onDebugResolve:NAMESPACE(edge)",
```

Comment:
Can we rename "Edge" to "Msedge" everywhere? In the old debugger, Edge has meant Legacy Edge while msedge is for the new Edge Version.  Would be good to continue that terminology in this as well.

Reply:
Done.